### ISSUE TITLE
Cannot view Scales with qpractice installed

### ISSUE THREAD

**Issue by drachels** [marcusgreen/moodle-mod_qpractice#2]
Trying to go to Site administration > Grades > Scales can't be done with qpractice module installed in Moodle 3.0.2+. It causes Field 'grade' does not exist in table 'qpractice' warning.
![viewscalecauseserror](https://cloud.githubusercontent.com/assets/1018249/13234788/1100b750-d980-11e5-8d65-81363ebd20bb.png)

**Comment by marcusgreen** [comment]
Thanks Al, I have just reproduced that error made a change to lib.php that should make that error go away. It looks like it was some code for a feature that was never implemented so I have put a return statement in as there is no support for the idea of scales. I have also made other minor changes so you might want to do a complete code refresh. It is mostly things like getting rid of redundant items and ensuring there are correct strings for everything.
